tests/lib/Microsoft.Fx.Portability.MetadataReader.Tests/TestAssembly.cs

```diff
@@ -5,38 +5,37 @@
 using Microsoft.CodeAnalysis.CSharp;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Reflection;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.Fx.Portability.MetadataReader.Tests
 {
     internal static class TestAssembly
     {
-        public static IAssemblyFile Create(string source, bool allowUnsafe = false)
-        {
-            return Create(source, allowUnsafe, Array.Empty<string>());
-        }
-
-        public static IAssemblyFile Create(string source, bool allowUnsafe, IEnumerable<string> additionalReferences)
+        private static readonly Assembly s_assembly = typeof(TestAssembly).GetTypeInfo().Assembly;
 
+        public static IAssemblyFile Create(string source, ITestOutputHelper output, bool allowUnsafe = false)
         {
-        switch (Path.GetExtension(source).ToLowerInvariant())
+            switch (Path.GetExtension(source).ToLowerInvariant())
             {
                 case ".dll":
                 case ".exe":
                     return new ResourceStreamAssemblyFile(source);
                 case ".cs":
-                    return new CSharpCompileAssemblyFile(source, allowUnsafe, additionalReferences);
+                    return new CSharpCompileAssemblyFile(source, allowUnsafe, Enumerable.Empty<string>());
```

Comment:
Not urgent, but it might be nice to write Roslyn diagnostics to output in this case since we'll always have an ITestOutputHelper anyhow.

Reply:
Also, do we not have a way to provide additional assembly reference for C# tests anymore?

Reply:
I can add that back, we just currently don't need it so I removed it

Reply:
Added the diagnostics to the output window